src/DynamoCoreWpf/ViewModels/Watch3D/Watch3DViewModelBase.cs

```diff
@@ -0,0 +1,439 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows.Media;
+using System.Xml;
+using Autodesk.DesignScript.Interfaces;
+using Dynamo.Core;
+using Dynamo.Interfaces;
+using Dynamo.Models;
+using Dynamo.Selection;
+using Dynamo.ViewModels;
+
+namespace Dynamo.Wpf.ViewModels.Watch3D
+{
+    public struct Watch3DViewModelStartupParams
+    {
+        public DynamoModel Model { get; set; }
+        public IRenderPackageFactory Factory { get; set; }
+        public DynamoViewModel ViewModel { get; set; }
```

Comment:
Maybe we can create a task to remove the hard dependency on `DynamoModel` and `DynamoViewModel`?